### DISCUSSION TITLE
Make an update of tooljet

### DISCUSSION THREAD

**Discussion by dedoface** [ToolJet/ToolJet#4846] [Q&A] [answered]
Dear,

I have a tooljet app uploaded in a Digital Ocean server. I have the 1.18.0 version and I want to upgrade to the new version to have all the feature.

I think, I can do it via npm package, but I have a fear, I will lost everything. Is there a way to make a proper update ?

Kind regards Adam.

**Comment by akshaysasidrn** [comment, marked as answer]
Hey @dedoface , you will have to make a new deployment with updated deploy.template.yaml pinning the version tag according to your requirement. In our template we have added postgres as a part of the template itself. You would want to move the db data to production setup that is db deployed separately and then have the tooljet server connect to it with `PG_*`  creds.

template example: https://github.com/ToolJet/ToolJet/blob/develop/.do/deploy.template.yaml#L9

**Comment by AlexandreSoftway** [comment]
hello! Thanks for the information, but how can I get a step by step to perform the update? I didn't understand how to update in self-host! and tried to find it in some documentation.
